src/components/Main/Results/AgeBarChart.tsx

```diff
@@ -4,16 +4,17 @@ import ReactResizeDetector from 'react-resize-detector'
 
 import { useTranslation } from 'react-i18next'
 
-import { Bar, BarChart, CartesianGrid, Legend, Tooltip, XAxis, YAxis, TooltipPayload } from 'recharts'
+import { Bar, BarChart, CartesianGrid, Legend, Tooltip, XAxis, YAxis } from 'recharts'
 
 import { AlgorithmResult } from '../../../algorithms/types/Result.types'
 
 import { SeverityTableRow } from '../Scenario/SeverityTable'
 
-import { numberFormatter } from '../../../helpers/numberFormat'
```

Comment:
The purpose of the numberFormatter is to localize, but also permit humanizing of displayed values (though this is currently broken in master, for me).

I'm not sure how `t('localized:number', { value: tick })` behaves for formatting, does this abstract the internationalization somehow?